server/server_internal_test.go

```diff
@@ -52,7 +55,7 @@ func TestHostName(t *testing.T) {
 			// Hostname set, no port in config, but port in url.  Expect <config_host>:<url_port>.
 			url:      "http://localhost:8080/capabilities",
 			hostName: "cdn.tegola.io",
-			expected: "cdn.tegola.io:8080",
+			expected: "cdn.tegola.io",
 		},
```

Comment:
I think there should be a test for setting the port and hostname. It would resolve in just the hostname, right?